vitrina/projects/views.py

```diff
@@ -93,4 +104,71 @@ class ProjectHistoryView(HistoryView):
     model = Project
     detail_url_name = 'project-detail'
     history_url_name = 'project-history'
-    tabs_template_name = 'component/tabs.html'
+    tabs_template_name = 'vitrina/projects/tabs.html'
+
+
+class ProjectDatasetsView(HistoryMixin, ListView):
+    model = Dataset
+    template_name = 'vitrina/projects/datasets.html'
+    paginate_by = 20
+
+    object: Dataset
+    detail_url_name = 'project-detail'
+    history_url_name = 'project-history'
+
+    def dispatch(self, request, *args, **kwargs):
+        self.object = get_object_or_404(Project, pk=kwargs['pk'])
+        return super().dispatch(request, *args, **kwargs)
+
+    def get_queryset(self):
+        return self.object.datasets.all()
```

Comment:
Use public manager to filter out deleted ant non-public datasets:

https://docs.djangoproject.com/en/3.2/topics/db/queries/#using-a-custom-reverse-manager